iothub/device/src/ModuleClient.cs

```diff
@@ -476,51 +480,51 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
         /// <param name="cancellationToken">A cancellation token to cancel the operation.</param>
         /// <exception cref="OperationCanceledException">Thrown when the operation has been canceled.</exception>
         public Task UpdateReportedPropertiesAsync(TwinCollection reportedProperties, CancellationToken cancellationToken) =>
-            this.internalClient.UpdateReportedPropertiesAsync(reportedProperties, cancellationToken);
+            InternalClient.UpdateReportedPropertiesAsync(reportedProperties, cancellationToken);
 
         #region Module Specific API
```

Comment:
this might need a comment explaining that this region differentiates between:
- APIs available in both device and module client
- APIs available only in module client.